stl/inc/map

```diff
@@ -516,6 +516,18 @@ void swap(multimap<_Kty, _Ty, _Pr, _Alloc>& _Left, multimap<_Kty, _Ty, _Pr, _All
     _Left.swap(_Right);
 }
 
+#if _HAS_CXX20
+template <class _Kty, class _Ty, class _Keylt, class _Alloc, class _Pr>
+void erase_if(map<_Kty, _Ty, _Keylt, _Alloc>& _Cont, _Pr _Pred) {

```

Comment:
I observe that this groups the `map` and `multimap` overloads together, instead of grouping the `map` overload with the rest of the `map` definition, as depicted in the Standard's header synopsis. (Ditto for the other `meow` and `multimeow` containers.)

It's true that the `pmr` aliases are grouped together below, but that's how they appear in the Standard's header synopsis.

Reply:
Actually, the *Standard* is inconsistent - the ordered containers don't group the `swap()` and `erase_if()` overloads together, but the unordered containers do. Our implementation currently doesn't group the unordered `swap()` together, so following that for `erase_if()` feels right. (I obsess too much about this.)

Reply:
I meant to double check that. I knew I forgot something.

I'm going to pretend that I looked at the unordered version and assumed they were the same :P